packages/pluggableWidgets/slider-web/src/Slider.editorConfig.ts

```diff
@@ -0,0 +1,146 @@
+    if (minValueType === "expression" && expressionMinimumValue === "") {
+        return {
+            property: "expressionMinimumValue",
+            message: "Minimum value expression is empty"
+        };
+    }
+
+    if (minValueType === "dynamic" && minAttribute === "") {
+        return {
+            property: "minAttribute",
+            message: "Missing attribute for 'Minimum value' property"
+        };
+    }
+};
+
+const maxValueCheck: CheckFn = ({ maxValueType, expressionMaximumValue, maxAttribute }) => {
+    if (maxValueType === "expression" && expressionMaximumValue === "") {
+        return {
+            property: "expressionMaximumValue",
+            message: "Maximum value expression is empty"
+        };
+    }
+
+    if (maxValueType === "dynamic" && maxAttribute === "") {
+        return {
+            property: "maxAttribute",
+            message: "Missing attribute for 'Maximum value' property"
+        };
+    }
+};
+
+const minMaxValueCheck: CheckFn = ({ minValueType, maxValueType, staticMinimumValue, staticMaximumValue }) => {
+    if (
+        minValueType !== "static" ||
+        maxValueType !== "static" ||
+        staticMaximumValue === null ||
+        staticMinimumValue === null ||
+        staticMinimumValue < staticMaximumValue
+    ) {
+        return;
+    }
+
+    return {
+        property: "staticMaximumValue",
+        message: "Maximum should be greater then Minimum value"
+    };
+};
+
+const tooltipCheck: CheckFn = values => {
+    if (values.showTooltip && values.tooltip === "") {
+        return {
+            property: "tooltip",
+            message: "Tooltip cannot be empty when 'Show tooltip' is enabled"
+        };
+    }
+};
+
+const stepSizeCheck: CheckFn = ({ stepSizeType, stepAttribute, stepValue, expressionStepSize }) => {
+    if (stepSizeType === "static" && stepValue !== null && stepValue <= 0) {
+        return {
+            property: "stepValue",
+            message: "Step size should be greater then zero"
+        };
+    }
+
+    if (stepSizeType === "expression" && expressionStepSize === "") {
+        return {
+            property: "expressionStepSize",
+            message: "Step size expression is empty"
+        };
+    }
+
+    if (stepSizeType === "dynamic" && stepAttribute === "") {
+        return {
+            property: "stepAttribute",
+            message: "Missing attribute for 'Step size' property"
+        };
+    }
+};
+
+const decimalPlacesCheck: CheckFn = ({ decimalPlaces }) => {
+    if (decimalPlaces !== null && decimalPlaces < 0) {
+        return {
+            property: "decimalPlaces",
+            message: "Decimal places cannot be negative"
+        };
+    }
+};
+
+export function check(values: SliderPreviewProps): Problem[] {
+    const checkers = [tooltipCheck, minMaxValueCheck, minValueCheck, maxValueCheck, stepSizeCheck, decimalPlacesCheck];
+
+    return checkers.map(checker => checker(values)).filter((p): p is Problem => !!p);
```

Comment:
I would prefer to have a more informative variable.
`p -> problem` or some name that explains the value.

Reply:
yeah, make sens